test/support/factory.ex

```diff
@@ -82,8 +82,8 @@ defmodule Oli.Factory do
 
   def community_admin_account_factory() do
     %CommunityAccount{
-      community: insert(:community),
-      author: insert(:author),
+      community: fn -> build(:community) end,
```

Comment:
I think you could just do `community: build(: community)`

Reply:
Don't want to answer for Mauricio by I believe the anon function is to force a lazy evaluation, which is the fix suggested by the people at ExMachina. That should prevent associating the same `Community` in each `CommunityAccount` when using `insert_list()`.